### PR TITLE
Port query_builder.e2e.spec.js to Cypress

### PR DESCRIPTION
**Pull request by tlrobinson**

### CODE REVIEW

File: frontend/test/metabase/scenarios/query_builder.cy.spec.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    });
+  });
+
+  describe("view mode", () => {
+    describe("summarize sidebar", () => {
+      it("should summarize by category and show a bar chart", () => {
+        cy.server();
+        cy.route("POST", "/api/dataset").as("dataset");
+        loadOrdersTable();
+        cy.wait("@dataset");
```
Comment (paulrosenzweig) [inline frontend/test/metabase/scenarios/query_builder.cy.spec.js]: Why did you need to wait on this API call for summarize but not filter?

**Review by paulrosenzweig — APPROVED** [review]
LGTM!

**Comment by paulrosenzweig** [comment]
FYI you can see videos of the CI failures under artifacts: https://app.circleci.com/jobs/github/metabase/metabase/184608/artifacts

**Review by kdoh — APPROVED** [review]

File: frontend/test/metabase/scenarios/query_builder.cy.spec.js (hunk 1 above)
Comment (tlrobinson) [reply frontend/test/metabase/scenarios/query_builder.cy.spec.js]: Honestly... I don't fully understand why it's necessary.

Comment (paulrosenzweig) [reply frontend/test/metabase/scenarios/query_builder.cy.spec.js]: I think I saw the issue when I throttled my network right before clicking "Orders". Something weird happens in the summarize sidebar. Maybe "Category" gets removed and readded quickly or something?

If we do need this wait, would `cy.contains("37.65")`  (or something else in the orders table) have the same effect as `cy.wait("@dataset")`? That'd be one line rather than three, but it's less clearly why we're doing it.
